src/app/backend/apihandler.go

```diff
@@ -140,6 +145,24 @@ func (apiHandler *ApiHandler) handleGetReplicaSetDetail(
 	response.WriteHeaderAndEntity(http.StatusCreated, result)
 }
 
+// Handles update of Replica Set API call.
+func (apiHandler *ApiHandler) handleUpdateReplicaSet(
+	request *restful.Request, response *restful.Response) {
+
+	replicaSetDetail := new(ReplicaSetDetail)
```

Comment:
Hmm... You've got name and namespace in the URL params? Which one should you use?